src/main/java/com/saralein/server/response/ResponseBuilder.java

```diff
@@ -0,0 +1,30 @@
+package com.saralein.server.response;
+
+public class ResponseBuilder {
+    private static String CRLF = "\r\n";
+
+    public static byte[] createResponse() {
```

Comment:
Can you talk about why you chose to return a byte array here?

Reply:
I picked bytes because, in my reading, it seemed to be what both sides understand.  Then picked byte array because that is what getBytes returns.  Also read about the content-length header being length of bytes.